advanced-examples/network-performance/network-performance-iperf3-k8s/README.md

```diff
@@ -39,116 +39,70 @@ $ arcaflow -input ${WFPATH}/input.yaml -config ${WFPATH}/config.yaml -context ${
 steps.kubeconfig.outputs.success-->steps.iperf3_client.deploy
 steps.kubeconfig.outputs.success-->steps.metadata.deploy
 steps.kubeconfig.outputs.success-->steps.pcp.deploy
-steps.service.deploy-->steps.service.starting
-steps.service.deploy-->steps.service.deploy_failed
-steps.metadata.deploy_failed-->steps.metadata.deploy_failed.error
-steps.kubeconfig.running-->steps.kubeconfig.outputs
-steps.kubeconfig.running-->steps.kubeconfig.crashed
+steps.uuidgen.deploy-->steps.uuidgen.starting
 steps.uuidgen.outputs.success-->outputs.success
-steps.pcp.starting-->steps.pcp.running
-steps.pcp.starting-->steps.pcp.crashed
-steps.iperf3_server.running-->steps.iperf3_server.outputs
-steps.iperf3_server.running-->steps.iperf3_server.crashed
-steps.uuidgen.deploy_failed-->steps.uuidgen.deploy_failed.error
-steps.service.outputs.success-->steps.iperf3_client.starting
-steps.iperf3_client.cancelled-->steps.iperf3_client.crashed
-steps.iperf3_client.cancelled-->steps.iperf3_client.deploy_failed
+steps.iperf3_client.starting-->steps.iperf3_client.running
+steps.iperf3_client.starting-->steps.iperf3_client.starting.started
 steps.iperf3_client.cancelled-->steps.iperf3_client.outputs
-steps.uuidgen.crashed-->steps.uuidgen.crashed.error
+steps.kubeconfig.outputs-->steps.kubeconfig.outputs.success
+steps.service.outputs-->steps.service.outputs.success
 steps.metadata.running-->steps.metadata.outputs
-steps.metadata.running-->steps.metadata.crashed
-steps.uuidgen.deploy-->steps.uuidgen.deploy_failed
-steps.uuidgen.deploy-->steps.uuidgen.starting
-steps.pcp.deploy_failed-->steps.pcp.deploy_failed.error
-steps.service.starting-->steps.service.running
-steps.service.starting-->steps.service.crashed
-steps.uuidgen.running-->steps.uuidgen.outputs
-steps.uuidgen.running-->steps.uuidgen.crashed
-steps.service.crashed-->steps.service.crashed.error
-steps.metadata.outputs-->steps.metadata.outputs.error
+steps.pcp.outputs-->steps.pcp.outputs.success
+steps.service.running-->steps.service.outputs
+steps.iperf3_client.running-->steps.iperf3_client.outputs
+steps.pcp.outputs.success-->outputs.success
+steps.iperf3_client.outputs.success-->outputs.success
+steps.uuidgen.outputs-->steps.uuidgen.outputs.success
+steps.pcp.deploy-->steps.pcp.starting
+steps.iperf3_server.outputs-->steps.iperf3_server.outputs.success
+steps.service.deploy-->steps.service.starting
+steps.kubeconfig.cancelled-->steps.kubeconfig.outputs
 steps.metadata.outputs-->steps.metadata.outputs.success
+steps.iperf3_server.cancelled-->steps.iperf3_server.outputs
+steps.kubeconfig.starting-->steps.kubeconfig.running
+steps.kubeconfig.starting-->steps.kubeconfig.starting.started
+steps.pcp.starting-->steps.pcp.starting.started
+steps.pcp.starting-->steps.pcp.running
+steps.iperf3_server.starting-->steps.iperf3_server.starting.started
+steps.iperf3_server.starting-->steps.iperf3_server.running
+steps.uuidgen.starting-->steps.uuidgen.starting.started
+steps.uuidgen.starting-->steps.uuidgen.running
```

Comment:
I noticed this browsing around in the diagram. Is this a "quirk" in the generator, or is these both real states? Note that `steps.uuidgen.starting` transitions to both `steps.uuidgen.running`, an intermediate state that transitions to `steps.uuidgen.outputs`, and also to `steps.uuidgen.starting.started`, a terminal state which seems to serve no obvious purpose, and seems a bit odd. All of the step "starting" states seem to do this.

Reply:
CC @jaredoconnell who is much more aware of the nuance of these states.  I believe `.started` is just a signaling mechanism that can be read by other steps, so if it is not read, it would make sense that it is a terminal point in the workflow.

Reply:
That makes sense.

Reply:
It's just the completed state of the starting phase. Nothing else within that workflow depends on the starting.started state.